setup.py

```diff
@@ -286,7 +286,8 @@ def run(self):
               '': ['*.zip', '*.svg', 'images/*'],
               'enable': ['tests/primitives/data/PngSuite/*.png'],
               'enable.savage.trait_defs.ui.wx': ['data/*.svg'],
-              'kiva': ['tests/agg/doubleprom_soho_full.jpg'],
+              'kiva': ['tests/agg/doubleprom_soho_full.jpg',
+                       'fonttools/tests/data/TestTTC.ttc'],
```

Comment:
Also the license file and source.txt (possibly should be doing that in other places as well, but don't worry about that in this PR).

Also make sure that the `MANIFEST.in` is catching these files as well.